python/ray/scripts/scripts.py

```diff
@@ -173,6 +173,11 @@ def dashboard(cluster_config_file, cluster_name, port):
     required=False,
     type=int,
     help="the port to use for starting the node manager")
+@click.option(
+    "--gcs-server-port",
```

Comment:
This name seems like it's exposing an implementation detail to users.

Reply:
Yeah I definitely agree. Indeed gcs server itself is already an implementation detail. I have two other proposal.

1. `gcs-manager-port`: This is aligning with the current port arguments.
2. Just having --min-ray-port --max-ray-port to assign Ray system ports. This will completely hide implementation details.

Reply:
I would be pretty happy with just having a single port range if we can pull that off

Reply:
I will work on that in the next sprint!